competitions/100Change2020/etl/deploy

```diff
@@ -43,6 +44,7 @@ if [ "${BASE_DATA_DIR}" = "" ]; then
   echo "                      list of keys to include.  If ARG begins with @,"
   echo "                      then ARG is a file with a newline separated list"
   echo "                      of keys to include."
+  echo "  -g                  Accepts a GPG phassphrase to use when decrypting."
```

Comment:
It would be good to specify that you mean symmetric passphrase here.  When people see "GPG passphrase", they usually think of the passphrase that protects the secret key of their public key pair.

Reply:
Good catch thank you!